.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main, clean-databricks-credentials]
```

Before merging, we should rollback this change:
```suggestion
    branches: [main]
```